docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -0,0 +1,212 @@
+import nvidia.dali as dali
+
+def input(file_root, annotations_file, device_id, num_threads, device, random_shuffle=True):
+    inputs, bboxes, classes = dali.fn.readers.coco(
+        file_root=file_root,
+        annotations_file=annotations_file,
+        ltrb=True,
+        shard_id=device_id,
+        num_shards=num_threads,
+        ratio=True,
+        random_shuffle=random_shuffle
+    )
+    images = dali.fn.decoders.image(inputs, device=device, output_type=dali.types.RGB)
+
+    return images, bboxes, classes
+
+def ltrb_to_xywh(bboxes):
+    Z = dali.types.Constant(0.0)
+    H = dali.types.Constant(0.5)
+    O = dali.types.Constant(1.0)
+    MO = dali.types.Constant(-1.0)
+
+    M = dali.fn.stack(
+            dali.fn.stack(H, Z, H, Z),
+            dali.fn.stack(Z, H, Z, H),
+            dali.fn.stack(MO, Z, O, Z),
+            dali.fn.stack(Z, MO, Z, O)
+        )
+    return dali.fn.coord_transform(bboxes, M=M)
+
+
+def bbox_adjust_xywh(bboxes, shape_x, shape_y, pos_x, pos_y):
+    Z = dali.types.Constant(0.0)
+
+    M = dali.fn.stack(
+            dali.fn.stack(shape_x, Z, Z, Z),
+            dali.fn.stack(Z, shape_y, Z, Z),
+            dali.fn.stack(Z, Z, shape_x, Z),
+            dali.fn.stack(Z, Z, Z, shape_y)
+        )
+
+    T = dali.fn.stack(pos_x, pos_y, Z, Z)
+    return dali.fn.coord_transform(bboxes, M=M, T=T)
```

Comment:
Assuming that the arguments can be DataNodes:
```suggestion
    Z = dali.types.Constant(0.0)

    scale = fn.transforms.scale(scale = fn.stack(shape_x, shape_y, shape_x, shape_y))
    MT = fn.transforms.translate(scale, offset = fn.stack(po_x, pos_y, 0, 0))
    return dali.fn.coord_transform(bboxes, MT=MT)
```
otherwise:
```suggestion
    MT = [shape_x, 0, 0, 0, pos_x,
         0, shape_y,, 0, 0, pos_y,
         0, 0, shape_x, 0, 0,
         0, 0, 0, shape_y, 0 ]
    return dali.fn.coord_transform(bboxes, MT=MT)
```

Reply:
I've removed this method, since it is no longer used in the code.